mmv1/third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -190,6 +190,9 @@ in Terraform state, a `terraform destroy` or `terraform apply` that would delete
 * `json_options` (Optional) - Additional properties to set if
     `source_format` is set to "JSON". Structure is [documented below](#nested_json_options).
 
+* `json_extension` (Optional) - Load option to be used together with to set if
```

Comment:
update docs to match what's in the API docs: https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#jsonextension

Reply:
done